provision/filter_plugins/gcloud_inventory.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
+
+from ansible import errors
+try:
+    from configparser import ConfigParser
+except ImportError:
+    from ConfigParser import ConfigParser
+import StringIO
+
+def pop_items(item_list,count):
```

Comment:
agreed . pop_items needs to be deleted . Yes , There is common functions , i was skeptical about the imports inside a filter file . Will try to make a simple library file from which we can import .